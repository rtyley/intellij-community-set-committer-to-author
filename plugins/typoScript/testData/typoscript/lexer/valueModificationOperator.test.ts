value1:=addToList(4,5)
value2:=addToList(4,5
value3:=( 4 and some text   )
value4:
value5:=add To My List ( 4,5)
value6:=add To My List(4,5)  some IGNORED_TEXT
value7: addToList( ,5)