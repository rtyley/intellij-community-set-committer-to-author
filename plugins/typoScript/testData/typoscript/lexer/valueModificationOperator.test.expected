object path entity ('value1')
:= (':=')
modification function ('addToList')
( ('(')
modification value ('4,5')
) (')')
WHITE_SPACE_WITH_NEW_LINE ('
')
object path entity ('value2')
:= (':=')
modification function ('addToList')
( ('(')
modification value ('4,5')
WHITE_SPACE_WITH_NEW_LINE ('
')
object path entity ('value3')
:= (':=')
( ('(')
WHITE_SPACE (' ')
modification value ('4 and some text')
WHITE_SPACE ('   ')
) (')')
WHITE_SPACE_WITH_NEW_LINE ('
')
object path entity ('value4')
BAD_CHARACTER (':')
WHITE_SPACE_WITH_NEW_LINE ('
')
object path entity ('value5')
:= (':=')
modification function ('add To My List')
WHITE_SPACE (' ')
( ('(')
WHITE_SPACE (' ')
modification value ('4,5')
) (')')
WHITE_SPACE_WITH_NEW_LINE ('
')
object path entity ('value6')
:= (':=')
modification function ('add To My List')
( ('(')
modification value ('4,5')
) (')')
IGNORED_TEXT ('  some IGNORED_TEXT')
WHITE_SPACE_WITH_NEW_LINE ('
')
object path entity ('value7')
BAD_CHARACTER (':')
WHITE_SPACE (' ')
object path entity ('addToList')
( ('(')
IGNORED_TEXT (' ,5)')